src/meta/app/src/schema/index.rs

```diff
@@ -67,6 +67,11 @@ pub struct IndexMeta {
     pub sync_creation: bool,
 }
 
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Eq, PartialEq, Default)]
+pub struct MarkedDeletedIndexMeta {
+    pub indexes: Vec<(u64, IndexMeta)>,
```

Comment:
It seems that most fields in `IndexMeta` is not needed, we just need the `index_id` and `drop_time`, we can add a new struct `DropIndexMeta` instead.
And it's better to add a `DropIndexType` field so that we can easily support other similar index type like `inverted index` and `virtual column`.
for example:
```rust
pub enum DropIndexType {
    AGGREGATING = 1,
    INVERTED = 2,
    VIRTUAL_COLUMN = 3,
}
pub struct DropIndexMeta {
    pub table_id: MetaId,
    pub index_id: MetaId,
    pub index_type: DropIndexType,
    pub dropped_on: DateTime<Utc>,
    ... other fields
}
```